src/order/conditionally_complete_lattice.lean

```diff
@@ -695,6 +703,10 @@ is_lub_le_iff (is_lub_cSup' hs)
 lemma cSup_le' {s : set α} {a : α} (h : a ∈ upper_bounds s) : Sup s ≤ a :=
 (cSup_le_iff' ⟨a, h⟩).2 h
 
+theorem le_cSup_iff' {s : set α} {a : α} (h : bdd_above s) :
```

Comment:
Is there an obvious `Inf` variant of this one too?

Reply:
It already existed, it's right below.

Reply:
No, since such a lemma would only be valid in conditionally complete lattices with a top element, and this type class currently doesn't exist. Theres `cSup_le_iff` for general conditionally complete lattices.